tests/compat.py

```diff
@@ -14,6 +15,7 @@
 from rq_dashboard.cli import make_flask_app
 
 HTTP_OK = 200
+VALID_RQ_VERSION = '1.2.0'
```

Comment:
Please rename to VERSION_1_2 or something like this.
Also, please check what will happen if `rq_version.VERSION = '1.10.0'`

Reply:
Reworked.